docs: add registration part to collator docs (#411)
t3rn/t3rn: docs/t0rn-collator.md

> # Run t3rn Collator on Rococo testnet (t0rn)

Reviewer: ```suggestion
# Run a t0rn Testnet Collator
```

> This guide outlines the essential minimum of steps required to run a collator for t0rn - a release candidate of t3rn to Rococo testnet. Guide uses the `v1.1.0-rc.0` release to Rococo, please however always use the latest available version.

Reviewer: ```suggestion
This guide outlines the essential minimum of steps required to run a collator for t0rn - a release candidate of t3rn on the Rococo testnet. This guide uses the `v1.1.0-rc.0` release, however always use the latest available version.
```

  --key-type aura
```

> ## Get some t0rn from faucet 

Reviewer: ```suggestion
## Get Some T0RN Balance
```

Your Collator needs some funds to register on testnet.

> Go to the [t0rn testnet faucet](https://dev.net.t3rn.io/faucet/), insert your substrate address and get some t0rn to cover transaction costs.

Reviewer: ```suggestion
Go to the [t0rn testnet faucet](https://dev.net.t3rn.io/faucet/), insert your substrate address and get some T0RN to cover transaction costs.
```

## Register as a candidate

> 1. Go to [polkadot.js](https://polkadot.js.org/apps/#/explorer), switch to t0rn and connect your collator account using polkadot.js

Reviewer: ```suggestion
1. Go to the [polkadot.js app](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fdev.net.t3rn.io#/accounts) and connect your collator account by clicking "Add account", then inserting your previously generated secret phrase aka mnemonic.
```

> 2. Set your session keys with your connected collator account under

Reviewer: ```suggestion
2. Generate a new session key pair and obtain the corresponding public key. Copy it to your clipboard as it is needed in the next step.
```

```

> Developer/RPC calls/author --> rotateKeys

Reviewer: ```suggestion
Developer --> RPC calls --> author --> rotateKeys()
```

```

> 3. Set the Session Key for your collator under

Reviewer: ```suggestion
3. Set the session key for your collator under
```

```

> Developer/Extrinsics --> session -> setKeys(sr25519_pubkey, 0x1234)

Reviewer: ```suggestion
Developer --> Extrinsics --> session -> setKeys(sr25519_pubkey, 0x1234)
```

```

4. Now finally register your collator as candidate.

```

> Developer/Extrinsics --> collatorSelection -> registerAsCandidate

Reviewer: ```suggestion
Developer --> Extrinsics --> collatorSelection -> registerAsCandidate()
```
